pkgs/veridian/no-override-prefix-path.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/veridian-slang/build.rs b/veridian-slang/build.rs
+index 186e694..10b2dc2 100644
+--- a/veridian-slang/build.rs
++++ b/veridian-slang/build.rs
+@@ -43,7 +43,6 @@ fn build_slang(slang_src: &Path, slang_install: &Path) {
+ fn build_slang_wrapper(slang: &Path, wrapper_install: &Path) {
+     cmake::Config::new("slang_wrapper")
+         .profile("Release")
+-        .define("CMAKE_PREFIX_PATH", slang)
```

Comment:
I don't know what's this doing, but if it builds then I guess it's fine ;)

Reply:
It's explained very briefly in the `pkgs/veridian/default.nix`, above where the patch is declared. In short, the path is set up nicely by the derivation. Having the `build.rs` overwrite this path and remove other dependencies isn't very helpful.